src/components/Sidebar/index.jsx

```diff
@@ -20,58 +27,99 @@ function Sidebar({ page, framesInfo }) {
 
   const handleNewProjectModalOpen = ev => {
     ev.preventDefault();
+
+    setIsClickedNewPorject(true);
   };
 
   return (
-    <SidebarWrapper>
-      <div className="page">
-        <h3 className="title" key={page.id}>
-          {page.name}
-        </h3>
-        <Button
-          handleClick={handlePageSelectModalOpen}
-          size="small"
-          usingCase="gray"
-        >
-          페이지 재선택
-        </Button>
-      </div>
-      <div className="frame-list">
-        <div className="titles">
-          <h3 className="title">전체 변경 화면</h3>
-          <h3 className="title-number"> {framesInfo.frameCount}</h3>
-        </div>
-        <ul role="presentation" onClick={handleFrameClick}>
-          {framesInfo.map(frame => (
-            <li
-              key={frame.id}
-              id={frame.id}
-              className={`frame-name ${
-                selectedFrameId === frame.id ? "active" : ""
-              }`}
+    <>
+      {isClickedNewProject && (
+        <Modal>
+          <TextWrapper>
+            <h1 className="reversion-title">새 프로젝트를 비교하시겠어요?</h1>
+            <Description
+              className="reversion-description"
+              size="medium"
+              text="비교하기 버튼을 누루면 현재 화면에서 벗어나게 됩니다.\n보고계신 정보는 저장되지 않아요."
```

Comment:
P1: 누르면으로 오타 수정 부탁드려요.

```suggestion
              text="비교하기 버튼을 누르면 현재 화면에서 벗어나게 됩니다.\n보고계신 정보는 저장되지 않아요."
```

Reply:
앗...! 바로 수정하겠습니다